### PR TITLE
pacman-files: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For: #5256

### CODE REVIEW

File: pages/linux/pacman-files.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Update the package database:
+
+`sudo pacman --files --refresh`
+
+- Find the package that owns a specific file:
+
+`pacman --files {{filename}}`
+
+- Find the package that owns a specific file using a regular expression:
```
Comment (bl-ue) [inline pages/linux/pacman-files.md:18]:
```suggestion
- Find the package that owns a specific file, using a regular expression:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

File: pages/linux/pacman-files.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`pacman --files --help`
+
+- Update the package database:
+
+`sudo pacman --files --refresh`
+
+- Find the package that owns a specific file:
+
+`pacman --files {{filename}}`
```
Comment (sbrl) [inline pages/linux/pacman-files.md:16]: Is this is filename only, or does it support a `path/to/file`?

Comment (navarroaxel) [reply pages/linux/pacman-files.md:16]: as a user you can user the `filename` or the exact full path to file like `usr/bin/git` and the output if different:
```
$  pacman -F git
 extra/git 2.30.1-1 [installed]
    usr/bin/git
    usr/lib/git-core/git
    usr/share/bash-completion/completions/git
    usr/share/git/mw-to-git/bin-wrapper/git
 pacacommunity/drupal 9.0.6-2
    usr/share/webapps/drupal/core/tests/Drupal/Tests/Composer/Plugin/Scaffold/fixtures/scripts/disable-git-bin/git
$  pacman -F usr/bin/git
usr/bin/git is owned by extra/git 2.30.1-1
```

Comment (bl-ue) [reply pages/linux/pacman-files.md:16]: I think it's better to keep it as "filename" 👍🏻

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

**Pull request merged**